README.md

```diff
@@ -321,44 +321,44 @@ returns a promise rejection.  The only exception to that behaviour is if the
 handler does not provide the `applyMethod` optimization trap.  Then, its
 default implementation is
 ```js
-HandledPromise.applyFunction(HandledPromise.get(p, prop), args)
+Promise.applyFunction(Promise.get(p, prop), args)
 ```
 
 This expansion requires that the promise for the remote method be unnecessarily
 reified.
 
 For an unfulfilled handler, the trap's `target` argument is the unfulfilled
-handled promise, so that it can gain control before the promise is resolved.
+delegated promise, so that it can gain control before the promise is resolved.
 For a fulfilled handler, the method's `target` argument is the result of the
 fulfillment, since it is available.
 
-### `HandledPromise` static methods
+### `Promise` static methods
 
 The methods in this section are used to implement higher-level communication
 primitives, such as the `E` proxy maker.
 
 These methods are analogous to the `Reflect` API, but asynchronously invoke a
-handled promise's handler regardless of whether the target has resolved.  This
+delegated promise's handler regardless of whether the target has resolved.  This
 is necessary in order to allow pipelining of messages before the exact
-destination is known (i.e. after the handled promise is resolved).
+destination is known (i.e. after the delegated promise is resolved).
 
 ```js
-HandledPromise.get(target, prop); // Promise<result>
-HandledPromise.getSendOnly(target, prop); // undefined
+Promise.get(target, prop); // Promise<result>
```

Comment:
Is hanging these methods off `Promise` (instead of `HandledPromise` as in the previous draft) a potential source of web compatibility issues due to people already having monkey patched `Promise` for some reason?  (I could especially see this as an issue with `get`, which is an incredibly common choice of method name).

Reply:
Resolved. npi.